xline/src/server/command.rs

```diff
@@ -278,6 +278,7 @@ where
             res = sync_res;
         }
         self.persistent.flush_ops(ops)?;
+        self.kv_storage.mark_index_available(res.revision());
```

Comment:
The `mark_index_available` will panic. Shall we deal with index error in this way? WDYT? @rogercloud

Reply:
we can't recover it, all keys in `unavailable_cache` will be inserted into `index` first and then into `unavailable_cache`, so it is impossible to read none here

Reply:
Then why don't you use the `unreachable!` macro to replace the `panic!`?

Reply:
Fixed